Translation errors when compiling with KDE support enabled

Error from `make` output with compiling with KDE support enabled:

```
cantata/po/ja.po:7: nplurals = 1...
cantata/po/ja.po:2823: ...but some messages have 2 plural forms
cantata/po/ja.po:2830: a format specification for argument 1 doesn't exist in 'msgstr[0]'
cantata/po/ja.po:6532: a format specification for argument 1 doesn't exist in 'msgstr[0]'
/usr/bin/msgfmt: found 3 fatal errors
```

Steps to reproduce:

```
$ mkdir build
$ cd build
$ cmake .. -DENABLE_KDE=ON
$ make
$ sudo make install
```

Workaround: Comment out line 63 in `cantata/build/po/CMakeFiles/translations.dir/build.make`

I'm now getting the following output:
```
Scanning dependencies of target translations
cantata/po/ja.po:2822:21: syntax error
cantata/po/ja.po:2828:26: syntax error
cantata/po/ja.po:2833:21: syntax error
cantata/po/ja.po:2838:25: syntax error
cantata/po/ja.po:2843:23: syntax error
cantata/po/ja.po:2848:21: syntax error
cantata/po/ja.po:2853:19: syntax error
cantata/po/ja.po:2858:20: syntax error
cantata/po/ja.po:2863:23: syntax error
cantata/po/ja.po:2868:29: syntax error
cantata/po/ja.po:6521:18: syntax error
/usr/bin/msgfmt: found 11 fatal errors
make[2]: *** [po/CMakeFiles/translations.dir/build.make:63: translations] Error 1
make[1]: *** [CMakeFiles/Makefile2:442: po/CMakeFiles/translations.dir/all] Error 2
make: *** [Makefile:150: all] Error 2
```

--- a/po/ja.po
+++ b/po/ja.po
@@ -5,6 +5,7 @@
 #
 msgid ""
 msgstr ""
+"Plural-Forms: nplurals=1; plural=0;\n"
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2016-12-27 01:19+0900\n"

